### ISSUE TITLE
SetVariable doesn't store String value (render null)

### ISSUE THREAD

**Issue by jdevillard** [elsa-workflows/elsa-core#777]
Hello, 

I've seen that when using a SetVariable in the Designer (Elsa 2.0), I cannot set String variable.
Due to the EvaluateAsync in the IExpressionHandler

```
var converter = TypeDescriptor.GetConverter(returnType);
var value = converter.CanConvertFrom(typeof(string)) ? converter.ConvertFrom(expression) : default;
```

because the returnType of the Activity Property (Value) of the SetVariable Activity is an Object.

In LiteralHandler it's return null, and in the LiquidExpressionHandler it throws an exception on the StringExtensions.cs when executing converter.ConvertFromString(value)! (TypeConverter cannot convert from System.String.)

**Comment by jdevillard** [comment]
issue can be tested with this kind of definition.

```csharp
 var workflowDefinition = new WorkflowDefinition
            {
                Id = "1",
                DefinitionId = "SampleWorkflow",
                Version = 1,
                IsPublished = true,
                IsLatest = true,
                PersistenceBehavior = WorkflowPersistenceBehavior.Suspended,
                Activities = new[]
                {
                    new ActivityDefinition
                    {
                        ActivityId = "activity-1",
                        Type = nameof(WriteLine),
                        Properties = new List<ActivityDefinitionProperty>()
                        {
                            ActivityDefinitionProperty.Liquid(nameof(WriteLine.Text), "Hello World")
                        }
                    }, new ActivityDefinition
                    {
                        ActivityId = "activity-2",
                        Type = nameof(SetVariable),
                        Properties = new List<ActivityDefinitionProperty>()
                        {
                            ActivityDefinitionProperty.Liquid(nameof(SetVariable.VariableName), "variableName"),
                            ActivityDefinitionProperty.Liquid(nameof(SetVariable.Value), "Test Value")
                        }
                    },
                },
                Connections = new[]
                {
                    new ConnectionDefinition()
                    {
                        SourceActivityId = "activity-1",
                        TargetActivityId = "activity-2",
                        Outcome = "Done"
                    }
                }
            };

```

**Comment by sfmskywalker** [comment]
Fixed by @craigfowler via #901
